### PR TITLE
Rename proxy discovery to service discovery

### PR DESCRIPTION
**Pull request by nkryuchkov**

### CODE REVIEW

**Review by evanlinjin — APPROVED** [review]
Very nice.

File: pkg/skyenv/values.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 const (
 	DefaultTpDiscAddr          = "http://transport.discovery.skywire.skycoin.com"
 	DefaultDmsgDiscAddr        = "http://dmsg.discovery.skywire.skycoin.com"
-	DefaultProxyDiscAddr       = "http://proxy.discovery.skywire.skycoin.com"
+	DefaultServiceDiscAddr     = "http://service.discovery.skywire.skycoin.com"
```
Comment (jdknives) [inline pkg/skyenv/values.go]: Please change that to service.discovery.skycoin.com

File: pkg/skyenv/values.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ const (
 const (
 	TestTpDiscAddr          = "http://transport.discovery.skywire.cc"
 	TestDmsgDiscAddr        = "http://dmsg.discovery.skywire.cc"
-	TestProxyDiscAddr       = "http://proxy.discovery.skywire.cc"
+	TestServiceDiscAddr     = "http://service.discovery.skywire.cc"
 	TestRouteFinderAddr     = "http://routefinder.skywire.cc"
 	TestUptimeTrackerAddr   = "http://uptime-tracker.skywire.cc"
 	TestAddressResolverAddr = "http://address-resolver.skywire.cc"
```
Comment (jdknives) [inline pkg/skyenv/values.go]: New domain is `address.resolver.skywire.cc`

File: pkg/skyenv/values.go [hunk 3]
```diff
@@ -21,7 +21,7 @@ const (
 const (
 	TestTpDiscAddr          = "http://transport.discovery.skywire.cc"
 	TestDmsgDiscAddr        = "http://dmsg.discovery.skywire.cc"
-	TestProxyDiscAddr       = "http://proxy.discovery.skywire.cc"
+	TestServiceDiscAddr     = "http://service.discovery.skywire.cc"
 	TestRouteFinderAddr     = "http://routefinder.skywire.cc"
 	TestUptimeTrackerAddr   = "http://uptime-tracker.skywire.cc"
```
Comment (jdknives) [inline pkg/skyenv/values.go]: New domain here is `uptime.tracker.skywire.cc?`.

**Review by jdknives — CHANGES_REQUESTED** [review]

**Review by jdknives — APPROVED** [review]

**Pull request merged**